crates/aptos/src/move_tool/bytecode.rs

```diff
@@ -38,7 +38,7 @@ use std::{
 };
 use tempfile::NamedTempFile;
 
-const DISASSEMBLER_EXTENSION: &str = "mv.asm";
+const DISASSEMBLER_EXTENSION: &str = "mv.masm";
```

Comment:
Nit: I think this also has legacy assembler, can we continue to use `.asm` for that, only for the new one?

Reply:
Please don't forget to re-request review so it shows up in dashboards.